consensus/istanbul/backend/engine.go

```diff
@@ -379,6 +384,10 @@ func (sb *Backend) Prepare(chain consensus.ChainReader, header *types.Header) er
 		header.Time = big.NewInt(time.Now().Unix())
 	}
 
+	if err := writeEmptyIstanbulExtra(header); err != nil {
```

Comment:
Is there a specific reason for this function?  From a quick glance, seems like it's putting zero values into all of the `IstanbulExtra` fields, which will happen by default when the header is constructed.